chalice/chalicelib/__init__.py

```diff
@@ -0,0 +1,23 @@
+import os
+import uuid
+
+from typing import List
+
+
+def rand_uuid() -> str:
+    return str(uuid.uuid4())
+
+
+def get_mtx_paths(dir: str, mtx_suffix: str) -> List[str]:
+    """
+    Get all matrices file paths within a directory.
+    :param dir: Directory that contains matrix files.
+    :param mtx_suffix: Suffix for the matrix files.
+    :return: A list of mtx that ends with the specific suffix in the directory.
+    """
+    mtx_paths = []
+
+    for dname, _, fnames in os.walk(dir):
```

Comment:
I think you can use `pathlib` for this now, something like `pathlib.Path(dir).glob("**/*." + mtx_suffix)`. Up to you, but the `os.walk` syntax always looked super ugly to me.

Reply:
Sure. I will switch to use pathlib.